app/js/components/orderEntry/PatientDashboard.jsx

```diff
@@ -1,62 +0,0 @@
-import React from 'react';
```

Comment:
Not sure what am missing here. If we are consolidating `CSS` why were non-css related code affected in this?

Reply:
@zeze111 that would call for a separate ticket and pull request.

Reply:
okay